### ISSUE TITLE
Make page <title> match its <h1> 

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#842]
Part of #820

For non-search-results and non-component pages, the `<title>` content is just "Arclight". This isn't useful for accessibility or people who have lots of tabs open. 

Adjust the titles for these pages:

- [ ] Home page:  `<title>Archival Collections at Institution - Arclight</title>`
- [ ] Repositories:  `<title>Repositories - Arclight</title>`
   https://arclight-demo.projectblacklight.org/repositories
- [ ] Individual repository:  `<title>[repository name] - Arclight</title>`
   https://arclight-demo.projectblacklight.org/repositories/duke-rubenstein
